tests/test_application_summary.py

```diff
@@ -42,23 +54,41 @@ def test_serialise_application_summary():
     assert applications[1].last_edited is None
 
 
-def test_dashboard_route(flask_test_client, requests_mock):
-    requests_mock.get(
-        "http://application_store/applications?account_id=test-user",
-        text=TEST_APPLICATION_STORE_DATA,
+def test_dashboard_route(flask_test_client, mocker):
+    mocker.patch(
+        "app.default.data.get_local_data",
```

Comment:
As this is a mock anyway, should it mock the production level "get_data"?

Reply:
good spot 🔍